<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginDialog</class>
 <widget class="QDialog" name="LoginDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>306</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>登录窗口</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resource.qrc">
    <normaloff>:/new/index/taskbar</normaloff>:/new/index/taskbar</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>2</x>
     <y>140</y>
     <width>302</width>
     <height>189</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>25</number>
    </property>
    <property name="rightMargin">
     <number>25</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QLineEdit" name="serverLineEdit">
      <property name="minimumSize">
       <size>
        <width>250</width>
        <height>50</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>230</width>
        <height>50</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
border: 0px groove solid;
</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="usernameLineEdit">
      <property name="minimumSize">
       <size>
        <width>250</width>
        <height>50</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>230</width>
        <height>50</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
border: 0px groove solid white;border-radius:4px
</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="Line" name="line_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget" native="true">
      <property name="minimumSize">
       <size>
        <width>250</width>
        <height>50</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>250</width>
        <height>50</height>
       </size>
      </property>
      <widget class="QLineEdit" name="passwdLineEdit">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>231</width>
         <height>50</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>50</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);
border: 0px groove solid white;border-radius:4px</string>
       </property>
      </widget>
      <widget class="QPushButton" name="loginPushButton">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>20</y>
         <width>18</width>
         <height>10</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>18</width>
         <height>10</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
	border-image: url(:/new/index/image/enter.png);
}
QPushButton::hover {
border-image: url(:/new/index/image/enter-hover.png);
}
</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="Line" name="line_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalWidget" native="true">
   <property name="geometry">
    <rect>
     <x>2</x>
     <y>2</y>
     <width>302</width>
     <height>31</height>
    </rect>
   </property>
   <widget class="QPushButton" name="closeButton">
    <property name="geometry">
     <rect>
      <x>267</x>
      <y>9</y>
      <width>24</width>
      <height>24</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>24</width>
      <height>24</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>24</width>
      <height>24</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/new/index/close_hover);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="widget_2" native="true">
   <property name="geometry">
    <rect>
     <x>2</x>
     <y>32</y>
     <width>302</width>
     <height>111</height>
    </rect>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>5</y>
      <width>130</width>
      <height>69</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>130</width>
      <height>69</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>130</width>
      <height>69</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/new/index/image/titlelogo.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="log_fail_warning_msg">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>241</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="widget_3" native="true">
   <property name="geometry">
    <rect>
     <x>2</x>
     <y>325</y>
     <width>302</width>
     <height>71</height>
    </rect>
   </property>
   <widget class="QCheckBox" name="checkBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>16</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
QCheckBox::indicator:unchecked {
border-image: url(:/new/index/image/checkbutton.png);
} 
QCheckBox::indicator::checked {
border-image: url(:/new/index/image/checkbutton_click.png);
}</string>
    </property>
    <property name="text">
     <string>记住密码</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>20</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(108, 108, 108);</string>
    </property>
    <property name="text">
     <string>forget password</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
